packages/upset/src/components/Columns/SizeBar.tsx

```diff
@@ -56,7 +56,7 @@ export const SizeBar: FC<Props> = ({ row, size }) => {
       return bookmarkedColorPallete[row.id];
     }
 
-    if (row !== undefined && currentIntersection !== null && currentIntersection.id === row.id) { // if currently selected, use the highlight colors
+    if (currentIntersection?.id === row?.id) { // if currently selected, use the highlight colors
```

Comment:
Is there a case where these can fail due to undefined or null pointer errors?

Reply:
Actually, now that I think about it, if both `row` and `currentIntersection` are `null` or `undefined`, the optional chaining operator will evaluate both to `undefined` and that `===` will return `true`. I'll add a check so that we don't have that happening.